docs/semgrep-app/pricing-and-billing.mdx

```diff
@@ -27,24 +27,32 @@ Semgrep App has three tiers:
 | Tier | Price | Description |
 | ---- | ------- | -------- |
 | **Community** | Free | For general-purpose security scanning.  |
-| **Team** | US$40 monthly per seat | For the enforcement of company-specific coding standards, private rules, and the analysis of findings.  |
+| **Team** | US$40 monthly per developer | For the enforcement of company-specific coding standards, private rules, and the analysis of findings.  |
```

Comment:
(Feel free to resolve after you read this comment) Just so I can understand r2c terminology: I guess that there is some business logic decision behind this renaming... just not sure that every user of Semgrep is a developer. "Contributor" looked fine as well. Let me know if there is some thread I can read about it or just ignore this comment. :)